api/v1alpha1/inferencepool_types.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
-// LLMServerPoolSpec defines the desired state of LLMServerPool
-type LLMServerPoolSpec struct {
+// InferencePoolSpec defines the desired state of InferencePool
+type InferencePoolSpec struct {
 
 	// ModelServerSelector uses a map of label to watch model server pods
-	// that should be included in the LLMServerPool. ModelServers should not
-	// be with any other Service or LLMServerPool, that behavior is not supported
+	// that should be included in the InferencePool. ModelServers should not
+	// be with any other Service or InferencePool, that behavior is not supported
 	// and will result in sub-optimal utilization.
 	// Due to this selector being translated to a service a simple map is used instead
```

Comment:
```suggestion
	// In some cases, implementations may translate this to a Service selector, so this matches the simple
	// map used for Service selectors instead of the full Kubernetes LabelSelector type.
```